pjmedia/src/pjmedia/stream.c

```diff
@@ -2453,7 +2455,7 @@ PJ_DEF(pj_status_t) pjmedia_stream_create( pjmedia_endpt *endpt,
 
     PJ_ASSERT_RETURN(endpt && info && p_stream, PJ_EINVAL);
 
-    if (pool == NULL) {
+    if (1 || pool == NULL) {
```

Comment:
what is this for?

Reply:
To make sure stream creating & using its own pool, as otherwise the premature destroy issue may still occur.